content/md/what-is/understand-data-quality.md

```diff
@@ -4,36 +4,26 @@ themes: first-steps
 title: Understand **data quality**
 popular: false
 ee-only: true
-related: improve-data-quality
+related: improve-data-quality, manage-data-quality, access-rights-on-data-quality
 ---
 
-# About data quality
-What is this **red, orange or green A, B, C, D, E displayed on my product edit form**? What are the **enrichment** and **consistency** grades?
+# About data quality insights
+What is this **red, orange or green A, B, C, D, E displayed on my product edit form**?
 
-**This is data quality insights!**
+![Image quality score](../img/quality-score.png)
+
+**This is the quality score calculated thanks to Akeneo's data quality insights feature!**
 
 Because rich, coherent and consistent product data is important in order to deliver great product experiences, we introduced the concept of "data quality" in our PIM.  
 This new capability helps you measure your data quality along two axes: **enrichment** and **consistency**.  
-Each of these two axes has a score from A to E. The scores can be seen on several PIM screens.
-
-::: tips
-By disabling specific attribute groups in Akeneo settings, you can focus your enrichment and consistency status and progress on the most relevant product data for your end-customer needs. Therefore, you will get insights on what’s truly important to improve your Product Experience.
-:::
-
-# How is enrichment calculated?
-The **enrichment grade** is calculated as the percentage of attributes with values in your product edit form (whether the attributes are required for completeness or not).
-
-We now know that an image is a must-have for a compelling product description. Our enrichment grade ensures that all your products have at least one image for your customers to see.
-
-Checking the `data quality insights` panel in the product edit form allows you to quickly see which attributes need to be filled in to improve your data quality grade for enrichment.
+Each of these two axes take into account several criteria to come up with an overall **quality score** for each product of your catalog.
```

Comment:
```suggestion
Each of these two axes takes into account several criteria to generate an overall **quality score** for each product of your catalog.
```

Reply:
not sure about 'overall' -> I would say a specific (because it's specific to each product individually)